docs/src/.vitepress/config.mts

```diff
@@ -4,6 +4,27 @@ import mathjax3 from "markdown-it-mathjax3";
 import footnote from "markdown-it-footnote";
 import { transformerMetaWordHighlight } from '@shikijs/transformers';
 
+function getBaseRepository(base: string): string {
+  if (!base) return '/';
+  // I guess if deploy_url is available. From where do I check this ?
+  const parts = base.split('/').filter(Boolean);
+  return parts.length > 0 ? `/${parts[0]}/` : '/';
+}
+
+const baseTemp = {
+  base: 'REPLACE_ME_DOCUMENTER_VITEPRESS',// TODO: replace this in makedocs!
```

Comment:
@asinghvi17 by the way, the way these strings are replaced makes the code unnecessarily ugly. Couldn't you give every bit its own name like `REPLACE_ME_DOCUMENTER_VITEPRESS_FAVICON`?

Reply:
We could, it's not super complicated either.  Will look into it today.